app/models/info_request.rb

```diff
@@ -925,6 +928,20 @@ def self.old_unclassified_params(extra_params, include_last_response_time=false)
     return params
   end
 
+  def self.old_unclassified_params(extra_params, include_last_response_time=false)
+    age = extra_params[:age_in_days] ? extra_params[:age_in_days].days : OLD_AGE_IN_DAYS
+    params = { :conditions => ["awaiting_description = ?
+                                    AND last_public_response_at < ?
+                                    AND url_title != 'holding_pen'
+                                    AND last_public_response_at IS NOT NULL
+                                    AND user_id IS NOT NULL",
+                                      true, Time.now - age] }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.